test/close-test.js

```diff
@@ -1,9 +0,0 @@
-const test = require('tape')
-const testCommon = require('./common')
-const abstract = require('abstract-leveldown/test/close-test')
-
-test('setUp', testCommon.setUp)
```

Comment:
Hmm, we should add a setup+teardown to https://github.com/Level/abstract-leveldown/blob/master/test/close-test.js

Reply:
And an `.all()` method.

Reply:
I'll make an issue and hack it up.